examples/ai-agent-orchestrator/scripts/pull-models.sh

```
@@ -0,0 +1,108 @@
+    local model=$1
+    curl -s "${OLLAMA_URL}/api/tags" | grep -q "\"name\":\"${model}\""
+}
+
+# Function to pull model
+pull_model() {
+    local model=$1
+    echo -e "${BLUE}📥 Pulling model: ${model}${NC}"
+    echo -e "${YELLOW}   This may take several minutes depending on model size...${NC}"
+    
+    # Use curl to call Ollama API
+    response=$(curl -s -X POST "${OLLAMA_URL}/api/pull" \
```

Shellcheck (0.11.0)

[warning] response appears unused. Verify use (or export if used externally).

(SC2034)